Service/ZGWService.php

```diff
@@ -278,32 +275,53 @@ public function downloadInhoudHandler(array $data, array $configuration): array
             $objectEntity instanceof ObjectEntity &&
             $objectEntity->getEntity()->getId()->toString() == $configuration['enkelvoudigInformatieObjectEntityId']
         ) {
+//            var_dump($objectEntity->getValueObject('inhoud')->getFiles()->first()->getBase64());
+
             $this->data['response'] = new Response(\Safe\base64_decode($objectEntity->getValueObject('inhoud')->getFiles()->first()->getBase64()), 200, ['content-type' => $objectEntity->getValueObject('inhoud')->getFiles()->first()->getMimeType()]);
         }
 
         return $this->data;
     }
 
-//    public function uploadFilePartHandler(array $data, array $configuration): array
-//    {
-//        $this->data = $data;
-//
-//        $parameters = $this->data;
-//        $pathDefintion = $this->data['path'];
-//        $path = array_combine($pathDefintion, explode('/', $parameters->getPathInfo()));
-//        $objectEntity = $this->entityManager->getRepository('App:ObjectEntity')->find($path['{id}']);
-//
-//        if($objectEntity->getEntity()->getId()->toString() !== $configuration['enkelvoudigInformatieObjectEntityId']) {
-//            return $this->data;
-//        }
-////          @TODO: Uncomment this once lock and release are proven to work
-////        if(!$objectEntity->toArray()['lock'] !== $this->data['lock']) {
-////            throw new \HttpException('Lock not valid', 400);
-////        }
-//
-//        $file = $objectEntity->getValueObject('inhoud')->getFiles()->first();
-//        $file->setBase64($file->getBase64().$data['inhoud']);
-//        $file->setSize(mb_strlen($file->getBase64()));
-
-//    }
+    public function uploadFilePartHandler(array $data, array $configuration): array
```

Comment:
Might want to add docblocks to function

Reply:
Ah, just a little bit too quickly, I'll add them. They have been removed on a merge conflict.